app/controllers/idv/sessions_controller.rb

```diff
@@ -7,7 +7,16 @@ class SessionsController < ApplicationController
     def index
     end
 
+    def finance
+    end
+
     def create
+      self.idv_params = applicant_params.delete_if { |_key, value| value.blank? }
```

Comment:
I think the issue is with submitting blank values to an upstream vendor. IIRC some vendor APIs croak if they get a blank value.